webpack/assets/javascripts/react_app/components/common/EmptyState/EmptyStatePattern.js

```diff
@@ -12,18 +22,22 @@ const EmptyStatePattern = props => {
     header,
     description,
   } = props;
+  const iconName = `${upperFirst(icon)}Icon`;
+  const iconClass = pficons[iconName];
 
   return (
-    <PfEmptyState>
-      <PfEmptyState.Icon type={iconType} name={icon} />
-      <PfEmptyState.Title>{header}</PfEmptyState.Title>
-      <PfEmptyState.Info>{description}</PfEmptyState.Info>
-      {documentation && <PfEmptyState.Help>{documentation}</PfEmptyState.Help>}
-      {action && <PfEmptyState.Action>{action}</PfEmptyState.Action>}
-      {secondaryActions && (
-        <PfEmptyState.Action secondary>{secondaryActions}</PfEmptyState.Action>
-      )}
-    </PfEmptyState>
+    <EmptyState variant={EmptyStateVariant.xl}>
+      <EmptyStateIcon icon={iconClass} />
+      <Title headingLevel="h5" size="4xl">
+        {header}
+      </Title>
+      <EmptyStateBody>{description}</EmptyStateBody>
+      {action}
+      <EmptyStateSecondaryActions>
+        {documentation}
```

Comment:
Looking at the old implementation,
I think the documentation should be under the description, and not as an action
something like the `PfEmptyState.Info` but the newer implementation